cma/evolution_strategy.py

```diff
@@ -4591,6 +4602,13 @@ def fmin_con(objective_function, x0, sigma0,
     True
     >>> # al.logger.plot()  # plots the evolution of AL coefficients
 
+    >>> x, es = cma.evolution_strategy.fmin_con(
+    ...             cma.ff.sphere, 2 * [0], 1, g=lambda x: [y+1 for y in x], post_optimization=True,
+    ...             options={'seed': 1, 'verbose': -9})
```

Comment:
This should work for any seed. If we do not make specific assertions that only work with a specific seed, the seed can and should be omitted.

Reply:
I agree, I just set a seed to make sure that the result would always be the same.

This and the previous comment were applied in the 'removed seed in post-optimization test and added feasibility assertion' commit.